types/result.ts

```diff
@@ -0,0 +1,11 @@
+import { z } from "zod";
+
+export const ResultKindEnum = z.enum(["positive", "negative", "neutral"]);
```

Comment:
qui secondo me vengono bene dei nomi più espliciti, tipo `win`, `userWins`, `youWin` o cose del genere e idem per gli altri

farei anche lo stesso rename come in Move per avere const e type con lo stesso nome